content/docs/started/getting-started-minikube.md

```diff
@@ -233,12 +233,13 @@ The following steps will deploy Kubeflow components and start them on the Miniku
 The above installation may take a few minutes. At the end of the installation you should see:
 ```
 Access Kubeflow dashboard at http://localhost:8080/
-Access JupyterHub at http://localhost:8080/hub/
+Access Jupyter notebooks at http://localhost:8080/notebooks/
 ```
 
 ### Where to go next
-Now you can access the Kubeflow dashboard at http://localhost:8080/ and JupyterHub at http://localhost:8080/hub/.
-For JupyterHub, you'll be landing on a login page.
+Now you can access the Kubeflow dashboard at http://localhost:8080/ and Jupyter
+notebooks at http://localhost:8080/notebooks/.
+For Jupyter notebooks, you'll be landing on a login page.
```

Comment:
Is this still correct? This looks like JupyterHub specific instructions, since users will pick their image when creating the Notebook server instance.

Reply:
Thanks! Fixed.